akka-cluster-typed/src/main/scala/akka/cluster/ddata/typed/internal/ReplicatorBehavior.scala

```diff
@@ -149,6 +149,7 @@ import akka.util.Timeout
                 rsp match {
                   case chg: dd.Replicator.Changed[_] => subscriber ! JReplicator.Changed(chg.key)(chg.dataValue)
                   case del: dd.Replicator.Deleted[_] => subscriber ! JReplicator.Deleted(del.key)
+                  case del: dd.Replicator.Expired[_] => subscriber ! JReplicator.Expired(del.key)
```

Comment:
just for consistency, looks like it should be `exp` rather than `del`

```suggestion
                  case exp: dd.Replicator.Expired[_] => subscriber ! JReplicator.Expired(exp.key)
```